CrossPlatformDesktopProject/CrossPlatformDesktopProject/NPC/Projectiles/Boomerang/Boomerang.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace CrossPlatformDesktopProject.NPC
+{
+    class Boomerang : INpc
```

Comment:
We already have a Boomerang.cs class for Link's equipables, might want to change this class name to be something different so that they can be separated (i.e. GoriyaBoomerang.cs)